platform/solr/solr-factory/src/main/java/org/codice/solr/factory/SolrServerFactory.java

```diff
@@ -68,9 +82,9 @@
 
     public static final String[] DEFAULT_PROTOCOLS = new String[] {"TLSv1.1", "TLSv1.2"};
 
-    public static final String[] DEFAULT_CIPHER_SUITES = new String[] {
-            "TLS_DHE_RSA_WITH_AES_128_CBC_SHA", "TLS_DHE_RSA_WITH_AES_128_CBC_SHA",
-            "TLS_DHE_DSS_WITH_AES_128_CBC_SHA", "TLS_RSA_WITH_AES_128_CBC_SHA"};
+    public static final String[] DEFAULT_CIPHER_SUITES =
+            new String[] {"TLS_DHE_RSA_WITH_AES_128_CBC_SHA", "TLS_DHE_RSA_WITH_AES_128_CBC_SHA",
```

Comment:
We shouldn't be hardcoding the cipher suites. These are available in the system properties and we should be pulling from there. 'System.getProperty("https.cipherSuites")'